### ISSUE TITLE
Rails backports are deprecated (sinatra-contrib)

### ISSUE THREAD

**Issue by DannyBen** [sinatra/sinatra#1598]
I see [many backports related issues](https://github.com/sinatra/sinatra/search?q=backports&type=Issues), but none are recent or open, so I am posting this issue in case it is not already known.

I see some issues are calling for removal of the backports dependency - maybe this is an opportunity to reconsider?

### Problem

When `sinatra-contrib` is required, there is a message printed to STDERR:

```
Rails backports are deprecated
```

### Reproduction

Run this script.

```ruby
#!/usr/bin/env ruby
require 'bundler/inline'

gemfile do
  source "https://rubygems.org"
  gem 'sinatra', '2.0.8.1'
  gem 'sinatra-contrib', '2.0.8.1'
end
```

### References

- The deprecation commit from Feb 16, 2020  
   https://github.com/marcandre/backports/commit/e01f4dfbf0cc5bf76151e408905b3be31948f2fe

**Comment by jkowens** [comment]
Closed by #1411.

**Comment by DannyBen** [comment]
Will this fix be released soon?
